tests/functional/hardLimit/testHardLimit.js

```diff
@@ -0,0 +1,50 @@
+const assert = require('assert');
+const sinon = require('sinon');
+const uuid = require('uuid');
+
+const { MonitorDiskUsage } = require('../../../libV2/tasks');
+
+const { fillDir } = require('../../utils/v2Data');
+
+// eslint-disable-next-line func-names
+describe('Test MonitorDiskUsage hard limit', function () {
+    this.timeout(30000);
+    let task;
+    let path;
+
+    beforeEach(async () => {
+        path = `/tmp/diskusage-${uuid.v4()}`;
+        task = new MonitorDiskUsage();
+        await task.setup();
+        task._path = path;
+        task._enabled = true;
+    });
+
+    it('should trigger a database unlock if below the limit', async () => {
```

Comment:
very minor: could be more logical to put the lock test before the unlock test since unlock happens after lock